abb_bringup/launch/abb_moveit_no_rviz.launch.py

```diff
@@ -63,6 +63,10 @@ def launch_setup(context, *args, **kwargs):
         "abb_irb1200_5_90_moveit_config", "config/kinematics.yaml"
     )
 
+    joint_limits_yaml = {
+        "robot_description_planning": load_yaml("abb_irb1200_5_90_moveit_config", "config/joint_limits.yaml")
+    }
```

Comment:
I hardcoded the package name `abb_irb1200_5_90_moveit_config` since it's what is done in the rest of the launch file but I find it very confusing. We have a `moveit_config_package` parameter that is used to get the urdf _but_ everything else is hardcoded to the `abb_irb1200_5_90`? This means that as soon as a user sets the parameter to anything different this will stop working in very confusing ways?

Still, didn't address it to avoid blowing up the size of this PR but happy to do it in a followup.

Reply:
Yeah we'll need to switch to [moveit_config_builder](https://github.com/moveit/moveit_resources/blob/ef5836ddf4d6a7773cee895a675bfa852a3798f6/panda_moveit_config/launch/demo.launch.py#L9) instead of manually specifying these configs. That can be a follow up to the upstream repo.